students/14th/team_1/seungje/project_westagram/account/views.py

```diff
@@ -0,0 +1,87 @@
+from django.views import View
+from .models import Account
+import json
+from django.http import request,JsonResponse
+
+class SignUp(View):
+
+    def post(self, request):
+        data = json.loads(request.body) #{name:박승제,number:010-26,email:tmd@djk,password:12345}
+        data_key = list(data.keys())
+
+        if 'name' in data_key:
+            if Account.objects.all().filter(user_name=data['name']).exists():
+                return JsonResponse({"message":"중복된 이름"},status=400)
+
+            if 'number' in data_key:
+                if Account.objects.all().filter(user_number=data['number']).exists():
+                    return JsonResponse({"message":"중복된 번호"},status=400)
+
+                if 'email' in data_key:
+                    if '.' and '@' in data['email']:
+                        if Account.objects.all().filter(user_email=data['email']).exists():
```

Comment:
```suggestion
                        if Account.objects.filter(user_email=data['email']).exists():
```
`all()`을 사용하지 않고 `filter`메소드 만으로도 객체를 불러오실 수 있습니다 :)

Reply:
`exists()` gooood!